airmozilla/popcorn/views.py

```diff
@@ -51,11 +42,68 @@ def event_meta_data(request):
 
     assert response.status_code == 302, response.status_code
 
+    purl = urlparse(response.headers['location'])
```

Comment:
Comment missing that explains *why* you're doing this. It's because of the `?t=091u23490-123` thing that vid.ly adds, right? And the reason for doing that is because it prevents popcorn-editor to index the media by its video URL.